libs/ui/elements/src/lib/list/list.component.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  Input,
+  OnChanges,
+  SimpleChanges,
+  ViewChild,
+} from '@angular/core'
+import { TranslateService } from '@ngx-translate/core'
+import { DatasetDistribution } from '@geonetwork-ui/common/domain/model/record'
+import { MatPaginator, PageEvent } from '@angular/material/paginator'
+
+@Component({
+  selector: 'gn-ui-list',
+  templateUrl: './list.component.html',
+  styleUrls: ['./list.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
friendly reminder that we strive to have all our new components and standalone :slightly_smiling_face: thanks!